t/op/sselect.t

```diff
@@ -106,16 +106,21 @@ select ($tie, undef, undef, $tie);
 ok("no crash from select $numeric_tie, undef, undef, $numeric_tie");
 
 SKIP: {
-    require POSIX;
+    my $SKIP_CR = sub {
+        skip shift, 4;
+    };
+
+    eval 'require POSIX' or do {
```

Comment:
Why quotes and not braces?

Reply:
No good reason. I think I thought the braces form caused a broken test, but that’s probably a mistake. Updated.